scim/usersRecords.bal

```diff
@@ -0,0 +1,357 @@
+// Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# + schemas - URIs of the used SCIM schemas  
```

Comment:
Shall we add a description to these records. To define what they are used for

Reply:
Check following sample:
https://github.com/ballerina-platform/module-ballerina-jwt/blob/1586df7e324b7a71ea2c7fda35e95808d608a797/ballerina/jwt_commons.bal#L58